### PR TITLE
Add covid-19 demo instance

### PR DESCRIPTION
**Pull request by francbartoli**
Add an instance that would respond from /covid-19

### CODE REVIEW

File: services/pygeoapi_covid19/start.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# https://github.com/docker-library/elasticsearch/issues/111
+sudo sysctl -w vm.max_map_count=262144
```
Comment (justb4) [inline services/pygeoapi_covid19/start.sh]: Removed this line as it is ES-specific, not required here.

**Comment by justb4** [comment]
Thanks for this work @francbartoli !

**Comment by francbartoli** [comment]
> I think all the right changes have been made to introduce this new service. Only question I have: the OGR ESRIJSON Provider/Driver was not officially supported. See this [pygeoapi issue](https://github.com/geopython/pygeoapi/issues/137), a long discussion with tendency to develop a
> [separate ESRIJSON Provider](https://github.com/geopython/pygeoapi/issues/137#issuecomment-501678741) . Or has that been changed? All the beter if we could use the OGR Provider. I am by no means an ESRI FS expert. If authentication is not an issue and also bbox and query by id works we could go this (OGR) way...

I wasn't aware of this @justb4. I hacked that support with https://github.com/geopython/pygeoapi/pull/391 and all tests passed so I hope most of the concerns are gone

**Comment by francbartoli** [comment]
> Thanks for this work @francbartoli !

Thanks for your help here @justb4

**Pull request merged**